lab_1_classify_by_unigrams/main.py

```diff
@@ -23,13 +20,14 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+
     if not isinstance(text, str):
         return None
-    tokenized_list = []
-    for token in text.lower():
-        if token.isalpha():
-            tokenized_list.append(token)
-    return tokenized_list
+    tokens = []
```

Comment:
revert all changes in lab_1

Reply:
Done I guess

Reply:
no, you can check ```files changed ``` here
![image](https://github.com/user-attachments/assets/e9bb0da3-6a1d-4422-a9d6-f5efd18886c7)

Reply:
I think I've figured it out ._.